src/extensions/AdditiveColormap3DExtensions/MinimumIntensityProjectionExtension.js

```diff
@@ -0,0 +1,18 @@
+import BaseExtension from './BaseExtension';
+import rendering from './rendering-modes';
+import { RENDERING_MODES as RENDERING_NAMES } from '../../constants';
+
+/**
+ * This deck.gl extension allows for an additive colormap like viridis or jet to be used for pseudo-coloring channels with Minimum Intensity Projection in 3D.
+ */
+const MinimumIntensityProjectionExtension = class extends BaseExtension {
+  constructor(args) {
+    super(args);
+    this.opts.rendering = rendering[RENDERING_NAMES.MIN_INTENSITY_PROJECTION];
```

Comment:
I'm confused with where/what `this.opts.rendering` does?

Reply:
This is what the docs were referring to.  It contains the `glsl` code that is run in the loop.